sdk/dapp-kit/test/hooks/useConnectWallet.test.tsx

```diff
@@ -57,7 +64,37 @@ describe('useConnectWallet', () => {
 		await waitFor(() => expect(result.current.connectWallet.isSuccess).toBe(true));
 		expect(result.current.currentWallet).toBeTruthy();
 		expect(result.current.currentWallet!.name).toBe('Mock Wallet 1');
-		expect(result.current.currentWallet!.accounts).toHaveLength(1);
+		expect(result.current.accounts).toHaveLength(1);
+		expect(result.current.currentAccount).toBeTruthy();
+
+		act(() => {
+			unregister();
+		});
+	});
+
+	test('only Sui accounts from multi-chain wallets are connected', async () => {
+		const { unregister, mockWallet } = registerMockWallet({
+			walletName: 'Mock Wallet 1',
+			accounts: [createMockAccount(), createMockAccount({ chains: ['solana:mainnet'] })],
+		});
+
+		const wrapper = createWalletProviderContextWrapper();
+		const { result } = renderHook(
+			() => ({
+				connectWallet: useConnectWallet(),
+				accounts: useAccounts(),
```

Comment:
The `accounts` state will only include Sui accounts, but `currentWallet.accounts` can include Sui and non-Sui accounts hence the slight re-working of these tests